kafka-ui-react-app/src/components/ConsumerGroups/Details/ListItem.styled.ts

```diff
@@ -1,6 +1,30 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
+type PropsType = {
+  size?: 'small' | 'large';
+};
 export const ToggleButton = styled.td`
   padding: 8px 8px 8px 16px !important;
   width: 30px;
 `;
+
+export const TableHeaderConsumerCell = styled.th<PropsType>(
+  ({ size, theme: { table } }) => css`
+    font-family: Inter, sans-serif;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 16px;
+    letter-spacing: 0em;
+    text-align: left;
+    justify-content: start;
+    align-items: center;
+    background: ${table.th.backgroundColor.normal};
+    cursor: default;
+    color: ${table.th.color.normal};
+    padding: 4px 0 4px ${size && table.th.title.padding.size[size]};
```

Comment:
you can do the check and put the values here directly.

 we don't have to put some values in the `theme` because of a some condition. it makes the `theme` larger and over complicates some styling logic

`size == 'small'? '16px' :  '24px';`

Reply:
i removed table title styles from theme